Makefile

```diff
@@ -4,10 +4,8 @@ setup-dev: requirements.txt requirements_test.txt
 	pip install --upgrade -r requirements.txt -r requirements_test.txt
 	pre-commit install
 
-# As a work-around for a test failure in tests/libs/datasets/data_source_test.py::test_state_providers_smoke_test
-# run tests on a single process. The failure seems to happen on the server (which runs unittest) but not locally.
 unittest:
-	pytest -n 1 tests/
+	pytest -n 2 tests/
```

Comment:
Note that putting this back to `-n 2` made a pretty trivial difference (6:23 => 6:04 time for CI workflow to run)...  So if the parallelization causes us any headaches in the future, we should probably just nuke it... especially since we're shifting to maintenance mode. 🤷